apps/api/READ_REPLICA_CHANGES.md

```
@@ -0,0 +1,249 @@
+
+# Option 2: Using individual environment variables
+POSTGRES_READ_REPLICA_DB=plane_replica
+POSTGRES_READ_REPLICA_USER=plane_user
+POSTGRES_READ_REPLICA_PASSWORD=your_password
+POSTGRES_READ_REPLICA_HOST=replica.example.com
+POSTGRES_READ_REPLICA_PORT=5432
+```
+
+## Architecture Overview
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)